lib/ImageProcessor.js

```diff
@@ -58,25 +59,27 @@ class ImageProcessor {
         if ( config.exists("backup") ) {
             const backup = config.get("backup");
             backup.acl = backup.acl || acl;
+            backup.cacheControl = (backup.cacheControl !== undefined) ? backup.cacheControl : cacheControl;
             backup.bucket = backup.bucket || bucket;
 
             promise = promise
               .then(() => this.execBackupImage(backup, imageData))
-              .then((image) => this.fileSystem.putObject(image))
+              .then((image) => this.fileSystem.putObject(image, backup))
               .then(() => ( backup.move === true ) ? this.fileSystem.deleteObject(imageData) : Promise.resolve())
               .then(() => Promise.resolve(++processedImages));
         }
 
         if ( config.exists("reduce") ) {
             const reduce = config.get("reduce");
             reduce.acl = reduce.acl || acl;
+            reduce.cacheControl = (reduce.cacheControl !== undefined) ? reduce.cacheControl : cacheControl;
             reduce.bucket = reduce.bucket || bucket;
             reduce.jpegOptimizer = reduce.jpegOptimizer || jpegOptimizer;
             reduce.optimizerOptions = optimizerOptions;
 
             promise = promise
               .then(() => this.execReduceImage(reduce, imageData))
-              .then((image) => this.fileSystem.putObject(image))
+              .then((image) => this.fileSystem.putObject(image, reduce))
```

Comment:
`reduce` will have much information. I suggest `{cacheControl: reduce.cacheControl}` is enough to pass and makes clearly.